frontend/src/components/Home/LocationCard.tsx

```diff
@@ -11,20 +11,38 @@ const LocationCard = ({ photo, location }: Props): ReactElement => {
   const [isMobile, setIsMobile] = useState<boolean>(false);
   const useStyles = makeStyles({
     img: {
-      height: 250,
-      width: '100%',
+      objectFit: 'cover',
+      objectPosition: 'center',
+    },
+    imgCropper: {
+      clipPath: 'inset(1%)',
+      scale: 1.03,
+    },
+    imgContainer: {
+      overflow: 'hidden',
+    },
+    textBox: {
+      height: '61.75px',
```

Comment:
It's great to see your attention to detail to match the figma but in the future, try to use whole numbers for styling (e.g. 61 px) or maximum 1 decimal place (e.g. 61.8 px) in exceptional cases where finetuning is really needed. This ensures cleaner code and more predictable design.